src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagementGetTasksBySearchCriteriaConsumerTest.java

```diff
@@ -44,119 +43,26 @@ public RequestResponsePact executeSearchQuery200(PactDslWithProvider builder) th
             .headers(getTaskManagementServiceResponseHeaders())
             .matchHeader(AUTHORIZATION, AUTH_TOKEN)
             .matchHeader(SERVICE_AUTHORIZATION, SERVICE_AUTH_TOKEN)
-            .body(createSearchEventCaseRequest(), String.valueOf(ContentType.JSON))
+            .body(createSearchEventCaseRequest(), APPLICATION_JSON)
```

Comment:
please revert the logic in pact tests wherever we changed to troubleshoot the issue